PocketKit/Sources/Textile/Tags/Components/TagsCell.swift

```diff
@@ -0,0 +1,96 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import SwiftUI
+
+public struct TagsCell: View {
+    let tag: TagType
+    let tagAction: (TagType) -> Void
+
+    public init(tag: TagType, tagAction: @escaping (TagType) -> Void) {
+        self.tag = tag
+        self.tagAction = tagAction
+    }
+
+    public var body: some View {
+        HStack {
+            Text(tag.name)
+                .style(.tags.allTags)
+                .accessibilityIdentifier("all-tags")
+            Spacer()
+            if case .recent = tag {
+                RecentTag()
+            }
+        }
+        .contentShape(Rectangle())
+        .onTapGesture {
+            tagAction(tag)
+        }
+    }
+}
+
+struct RecentTag: View {
+    public var body: some View {
+        HStack(alignment: .center, spacing: 0) {
+            Image(asset: .tag)
+                .tagIconStyle()
+            Text("recent")
+                .style(.tags.tag)
+        }
+        .accessibilityIdentifier("recent-tags")
+    }
+}
+
+extension Image {
```

Comment:
Consider making private or some other way of locking down access to this property.